UserInputValidator.cs

```diff
@@ -2,21 +2,15 @@ namespace TictactoeVer2
 {
     public class UserInputValidator
     {
-        public bool IsBoardSizeValid(string input)
+        public bool TryParseBoardSize(string input, out int size)
```

Comment:
I like it! With this change, do you still feel that this class is purely a validator? Given that it now also returns a "clean" value, it might be due a name change.